feat(MKEA-164): add marketplace examples and templates (#1085)
mia-platform/documentation: docs/marketplace/add_to_marketplace/manage_marketplace_items.md

      }
    }
  },
  "supportedBy": "Acme Corporation Inc.",
  "supportedByImage": {
    "localPath": "./acmeCorporation.png"
  },
  "tenantId": "acme-corporation",
  "type": "plugin"
}
```
</details>

You also want users to write services in your brand new programming language, `Acme.Js`.

> You want to provide a Template, which is a skeleton with a minimal setup: your users are required to write business code from scratch.

Reviewer: I would better introduce both Templates and Examples and why you should use them in this example. 
Maybe, it would also be possible to redirect to the specific docs pages concerning Templates and Examples (and even enrich also them)

  "description": "A simple Hello World example based on Acme Corporation Acme.Js Template.",
  "documentation": {
    "type": "markdown",
    "url": "https://raw.githubusercontent.com/acme-corporation/Acme-Js-example/master/README.md"
  },
  "image": {
    "localPath": "./acmeJsExample.png"
  },
  "itemId": "acme-js-example",
  "name": "TypeScript Hello World Example",
  "resources": {
    "services": {
      "acme-js-example": {
        "archiveUrl": "https://github.com/acme-corporation/Acme-Js-example/archive/master.tar.gz",
        "containerPorts": [
          {
            "from": 80,
            "name": "http",
            "protocol": "TCP",
            "to": 3000
          }
        ],
        "name": "acme-js-example",
        "type": "example"
      }
    }
  },
  "supportedBy": "Acme Corporation Inc.",
  "supportedByImage": {
    "localPath": "./acmeCorporation.png"
  },
  "tenantId": "mia-platform",
  "type": "example",
  "visibility": {
    "allTenants": false,
    "public": true
  }
}
```
</details>

Notice that the `image` and `supportedByImage` objects are populated with local paths to images: make sure the images exist and that their path is correct.

> To create the items on the Marketplace, open up a terminal in directory where the files are places and run this command:

Author: ```suggestion
To create the items on the Marketplace, open up a terminal in directory where the files are placed and run this command:
```

```sh
miactl marketplace apply -f myAwesomeService.json -f myAcmeJsTemplate.json -f myAcmeJsExample.json
```

> This command will create the Marketplace item and upload the images along with it.

Author: ```suggestion
This command will create the Marketplace items and upload the images along with them.
```

      }
    }
  },
  "supportedBy": "Acme Corporation Inc.",
  "supportedByImage": {
    "localPath": "./acmeCorporation.png"
  },
  "tenantId": "acme-corporation",
  "type": "plugin"
}
```
</details>

> You also want user to write services in your brand new programming language, `Acme.Js`.

Author: ```suggestion
You also want users to write services in your brand new programming language, `Acme.Js`.
```
